### PR TITLE
Use checkpoint signature for error reporting in sdk #1544

### LINKED ISSUE #1544: Use anonymous machine token from checkpoint for error reporting
## Context

In order to improve the product quality of Prisma, we're collecting [anonymous telemetry](https://github.com/prisma/prisma2/blob/bd1cad8e250dd73461bf2a2a663f3b2041d84b43/docs/telemetry.md) information which can be opted out at any time. Additionally when an unexpected error happens we're also prompting the user to optionally send a more detailed error report. Right now both cases are handled through separate systems which we want to unify to minimize internal technical complexity and better understand how many users are running into errors with Prisma.

## Proposal

As part of the telemetry setup, we're randomly generating a machine token which is used to identify an installation of Prisma. We also want to use this same machine token for the optional error report system.

### PR DESCRIPTION
**Pull request by Jolg42**

### CODE REVIEW

File: cli/sdk/src/sendPanic.ts [hunk 1]
```diff
@@ -9,6 +8,7 @@ import os from 'os'
 import path from 'path'
 import stripAnsi from 'strip-ansi'
 import tmp from 'tmp'
+import * as checkpoint from 'checkpoint-client'
```
Comment (timsuchanek) [inline cli/sdk/src/sendPanic.ts]: We can simplify this to `import checkpoint from 'checkpoint-client'` as we have `esModuleInterop: true` in our tsconfig.json.

**Review by timsuchanek — CHANGES_REQUESTED** [review]

File: cli/sdk/src/sendPanic.ts [hunk 2]
```diff
@@ -64,7 +64,7 @@ export async function sendPanic(
       platform: await getPlatform(),
       liftRequest: JSON.stringify(error.request),
       schemaFile: maskedSchema,
-      fingerprint: getFid() || undefined,
+      fingerprint: getFid(),
```
Comment (timsuchanek) [inline cli/sdk/src/sendPanic.ts]: We can directly call `checkpoint.signature.sync()` here

File: cli/sdk/package.json [hunk 3]
```diff
@@ -41,6 +41,7 @@
     "@prisma/get-platform": "workspace:*",
     "archiver": "3.0.0",
     "chalk": "^3.0.0",
+    "checkpoint-client": "^1.0.7",
```
Comment (timsuchanek) [inline cli/sdk/package.json]: Let's use the same version in both package.json's.
I suggest `^1.0.7` as we control that package and can trust the patches.

File: cli/sdk/src/sendPanic.ts (hunk 1 above)
Comment (Jolg42) [reply cli/sdk/src/sendPanic.ts]: Indeed!

File: cli/sdk/src/sendPanic.ts (hunk 2 above)
Comment (Jolg42) [reply cli/sdk/src/sendPanic.ts]: I was thinking about that too :)

File: cli/sdk/package.json (hunk 3 above)
Comment (Jolg42) [reply cli/sdk/package.json]: 👍

**Pull request merged**